### DISCUSSION TITLE
Database and Table context menu

### DISCUSSION THREAD

**Discussion by MatyCZ** [dbeaver/dbeaver#85] [Ideas]
Table context menu contains items:
- Export data
- Import data

But Database context menu contains similar items one level deeper:
- Tools > Export database
- Tools > Import database

Can you make these context menu more similar? Move subitems from Tools to root?

**Comment by serge-rider** [comment]
These items are doing very different things. 
Tools supposed to execute database-specific functions (usually provided by native cli tools). 
Number of elements in Tools is not predefined and can be extended by plugins. I don't think that moving them into root menu makes sense. It will make context menu quite trashy.

**Comment by serge-rider** [comment]
Although generally I agree that existing context menu could be improved.
